ColorPatternsTests/ColorPatternsTests.swift

```diff
@@ -7,8 +7,11 @@
 //
 
 import XCTest
+@testable import ColorPatterns
 
-class ColorPatternsTests: XCTestCase {
+final class ColorPatternsTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/4635961d82e0d70ae97fbe61ee7a7bdb9551efa2/ColorPatternsTests/ColorPatternsTests.swift#L12"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>